tools/bench/core/bench.h

```diff
@@ -0,0 +1,145 @@
+#ifndef PHARE_BENCH_CORE_BENCH_H
```

Comment:
should that be considered for this PR?

Reply:
it's not in src, and it's pretty nice so why not?